tensorflow/lite/examples/python/label_image.py

```diff
@@ -57,9 +55,15 @@ def load_labels(filename):
       '--input_std',
       default=127.5, type=float,
       help='input standard deviation')
+  parser.add_argument(
+      '--num_threads',
+      default=1,
```

Comment:
How about adding `type=int` here, so that line 66 can be just `num_threads=args.num_threads`?